src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -91,21 +91,28 @@ class SideNav extends Component {
     }
 
     return (
-      <ul
+      <div
+        id="va-sidenav-wrapper"
         className={`usa-width-one-fourth va-sidenav vads-u-margin--1 medium-screen:vads-u-height--auto medium-screen:vads-u-margin-y--0 medium-screen:vads-u-margin-left--0 medium-screen:vads-u-margin-right--2p5 ${
```

Comment:
This is code that was there previously, but if you want you could use the `classnames` library like we do here: https://github.com/department-of-veterans-affairs/vets-website/blob/5d789ffce88185c3c42e028c855320de4735a86b/src/platform/site-wide/side-nav/components/NavItemRow.js#L4

Reply:
:+1: